lib/web_ui/dev/goldens_lock.yaml

```diff
@@ -1,2 +1,2 @@
-repository: https://github.com/flutter/goldens.git
-revision: b6efc75885c23f0b5c485d8bd659ed339feec9ec
+repository: https://github.com/nturgut/goldens.git
```

Comment:
added to test this PR: https://github.com/flutter/goldens/pull/111

Reply:
Changed this to flutter repo, yet we still need to change the has once the goldens PR is in.

Reply:
made one more change for this. Hopefully they are the correct files both for chrome and ios-safari this time.